<?xml version="1.0"?>

<launch>
    <arg name="sim" default="true"/>
    <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++ -->

    <!-- Manage moveit scene -->
    <node name="manage_moveit_scene_node" pkg="grasp_pipeline"
        type="manage_moveit_scene_server.py" output="screen">
        <param name="sim" value="$(arg sim)"/>
    </node>

    <!-- Moveit planner node for panda arm -->
    <node name="arm_moveit_planner_server_node" pkg="grasp_pipeline" type="arm_moveit_planner_server.py" output="screen"/>

    <!-- Save visual data node -->
    <node name="save_visual_data_node" pkg="grasp_pipeline" type="save_visual_data_server.py" output="screen"/>
</launch>
